chainer/__init__.py

```diff
@@ -147,7 +147,11 @@ def is_arrays_compatible(arrays):
 global_config.use_ideep = os.environ.get('CHAINER_USE_IDEEP', 'never')
 global_config.lazy_grad_sum = bool(int(
     os.environ.get('CHAINER_LAZY_GRAD_SUM', '0')))
-global_config.cudnn_fast_batch_normalization = False
+global_config.cudnn_fast_batch_normalization = bool(int(
+    os.environ.get('CHAINER_CUDNN_FAST_BATCH_NORMALIZATION', '0')))
+if not global_config.cudnn_fast_batch_normalization:
+    global_config.cudnn_fast_batch_normalization = bool(int(
+        os.environ.get('CHAINER_CUDNN_FAST_BN', '0')))
```

Comment:
Why do you add this environment variable?

Reply:
Is your question about `CHAINER_CUDNN_FAST_BN`? That is because I thought `CHAINER_CUDNN_FAST_BATCH_NORMALIZATION` is bit too long and easy to make a mistake. But, at the same time, I also feel having two env-vars for the same purpose is rather redundant. What would you think?

Reply:
Thanks